modelon/impact/client/entities.py

```diff
@@ -23,6 +23,14 @@ def _assert_successful_operation(is_successful, operation_name="Operation"):
         )
 
 
+def _assert_unique_case_initialization(data, unsupported_init):
+    if data['input'][unsupported_init]:
+        raise ValueError(
+            "A case cannot use both 'initialize_from_case' and "
+            "'initialize_from_external_result' to specify what to initialize from!"
+        )
```

Comment:
It would be nicer if this was a metod on `Case` that took `self` instead of `data` as first argument.

Reply:
The error message should also say what the user can do to fix the error. The user can work around this by setting the other initialize from attribute to `None` and then re-try.